### PR TITLE
PAYARA-2539 Speed up intialisation of metrics boot

### PR DESCRIPTION
**Pull request by Cousjava**
Made it only initialise when enabled and also initialisation to happen on its own thread

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java [hunk 1]
```diff
@@ -107,7 +109,18 @@ public MetricsService() {
     public void init() {
         events.register(this);
         metricsServiceConfiguration = serviceLocator.getService(MetricsServiceConfiguration.class);
-        initMetadataConfig(getConfig(), false);
+        //only start if metrics are enabled
+        if (isMetricsEnabled()) {
+            
+            PayaraExecutorService payaraExecutor = serviceLocator.getService(PayaraExecutorService.class, new Annotation[0]);
+            payaraExecutor.submit(new Runnable () {
```
Comment (arjantijms) [inline appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java]: Good opportunity to use a lambda ;)

File: appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java [hunk 2]
```diff
@@ -107,7 +109,18 @@ public MetricsService() {
     public void init() {
         events.register(this);
         metricsServiceConfiguration = serviceLocator.getService(MetricsServiceConfiguration.class);
-        initMetadataConfig(getConfig(), false);
+        //only start if metrics are enabled
```
Comment (arjantijms) [inline appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java]: // Only start if metrics are enabled

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**